lgc/patch/PatchLoopMetadata.cpp

```diff
@@ -130,14 +169,49 @@ bool PatchLoopMetadata::runOnModule(Module &module) {
         Metadata *nonforcedMeta[] = {MDString::get(*m_context, "llvm.loop.disable_nonforced")};
         MDNode *nonforcedMetaNode = MDNode::get(*m_context, nonforcedMeta);
         loopMetaNode = MDNode::concatenate(loopMetaNode, MDNode::get(*m_context, nonforcedMetaNode));
+        changed = true;
+      } else if (!StrictUnrollHints) {
+        for (unsigned i = 1, e = loopMetaNode->getNumOperands(); i < e; ++i) {
+          Metadata *op = loopMetaNode->getOperand(i);
+          if (MDNode *mdNode = dyn_cast<MDNode>(op)) {
+            const MDString *mdString = dyn_cast<MDString>(mdNode->getOperand(0));
+            if (mdString)
```

Comment:
Can you move the definition to the if condition and add braces around the body?

Reply:
Done